feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/vif.c

```diff
@@ -361,13 +365,27 @@ static int vif_config_security_set(struct blob_buf *b,
 			{
 				blobmsg_add_u32(b, "acct_interval", acct_interval);
 			}
+
+			/*
+			 * If Radius is configured and Roaming is enabled,
+			 * - disable ft_psk_generate_local. This is required for hostapd to populate R0/R1.
+			 * - populate r0KH and r1KH with broadcast addressing so that the corresponding
+			 * Key Holders are auto-generated
+			 */
+			if (vconf->ft_mobility_domain) {
+				blobmsg_add_bool(b, "ft_psk_generate_local", 0);
+				blobmsg_add_string(b, "r0kh", "ff:ff:ff:ff:ff:ff,*,8a7fcc966ed0691ff2809e1f38c16996");
```

Comment:
What is this magic number  8a7fcc....  ?
What makes use of the content of ft_mobility_domain content?

Reply:
Removed from code.